shell/common/node_bindings.cc

```diff
@@ -180,6 +182,43 @@ bool AllowWasmCodeGenerationCallback(v8::Local<v8::Context> context,
   return node::AllowWasmCodeGenerationCallback(context, source);
 }
 
+v8::MaybeLocal<v8::Promise> HostImportModuleDynamically(
+    v8::Local<v8::Context> context,
+    v8::Local<v8::Data> v8_host_defined_options,
+    v8::Local<v8::Value> v8_referrer_resource_url,
+    v8::Local<v8::String> v8_specifier,
+    v8::Local<v8::FixedArray> v8_import_assertions) {
+  // If we're running with contextIsolation enabled in the renderer process,
+  // fall back to Blink's logic.
+  if (node::Environment::GetCurrent(context) == nullptr) {
+    if (gin_helper::Locker::IsBrowserProcess())
+      return v8::MaybeLocal<v8::Promise>();
```

Comment:
this should probably be `UNREACHED`, there should be no situation where we're in the browser process but don't have a node environment, right?

Reply:
I think this is a shutdown race condition we've seen before where `GetCurrent` doesn't return valid during shutdown 🤔 We have this check in lot's of these bindings helpers